src/DynamoCoreWpf/Views/Core/CustomColorPicker.xaml

```diff
@@ -1,358 +1,814 @@
-<Popup x:Class="Dynamo.Controls.CustomColorPicker"
-       xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
-       xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-       xmlns:xctk="http://schemas.xceed.com/wpf/xaml/toolkit"
-       xmlns:sys="clr-namespace:System;assembly=mscorlib"
-       xmlns:controls="clr-namespace:Dynamo.Controls"
-       xmlns:ui="clr-namespace:Dynamo.UI"
-       xmlns:p="clr-namespace:Dynamo.Wpf.Properties"
-       mc:Ignorable="d" 
-       d:DesignHeight="780" d:DesignWidth="860">
+<Popup
+    x:Class="Dynamo.Controls.CustomColorPicker"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="clr-namespace:Dynamo.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:p="clr-namespace:Dynamo.Wpf.Properties"
+    xmlns:sys="clr-namespace:System;assembly=mscorlib"
+    xmlns:ui="clr-namespace:Dynamo.UI"
+    xmlns:viewmodel="clr-namespace:Dynamo.ViewModels"
+    xmlns:xctk="http://schemas.xceed.com/wpf/xaml/toolkit"
+    x:Name="PART_ColorPickerPalettePopup"
+    Width="520"
+    Height="Auto"
+    VerticalAlignment="Bottom"
+    d:DesignHeight="550"
+    d:DesignWidth="800"
+    AllowsTransparency="True"
+    Focusable="False"
```

Comment:
what does this doe? I guess the focus will remain on Dynamo window?

Reply:
@QilongTang I changed this property to True so it will have the focus when the Popup is opened.
See the next commit: https://github.com/DynamoDS/Dynamo/pull/13824/commits/ada8c38823e02400415c4ab34c2c821b50b6904f